articles/hooks/extensibility-points/send-phone-message.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
This file will get linked from the Multi-Factor Authentication -> SMS -> Help text. Please see ULP-1734 for more info.

Reply:
👍